lcmtypes/BUILD.bazel

```diff
@@ -19,23 +19,23 @@ lcm_cc_library(
     tags = ["nolint"],
 )
 
-lcm_py_library(
-    name = "lcmtypes_rs2_py",
-    lcm_package = "rs2_lcm",
-    lcm_srcs = LCM_SRCS,
-    tags = ["nolint"],
-    deps = [
-        "@lcmtypes_robotlocomotion//:lcmtypes_robotlocomotion_py",
-    ],
-)
+# lcm_py_library(
+#     name = "lcmtypes_rs2_py",
+#     lcm_package = "rs2_lcm",
+#     lcm_srcs = LCM_SRCS,
+#     tags = ["nolint"],
+#     deps = [
+#         "@lcmtypes_robotlocomotion//:lcmtypes_robotlocomotion_py",
+#     ],
+# )
 
-lcm_java_library(
-    name = "lcmtypes_rs2_java",
-    lcm_package = "rs2_lcm",
-    lcm_srcs = LCM_SRCS,
-    deps = [
-        "@lcmtypes_robotlocomotion//:lcmtypes_robotlocomotion_java",
-    ],
-)
+# lcm_java_library(
+#     name = "lcmtypes_rs2_java",
+#     lcm_package = "rs2_lcm",
+#     lcm_srcs = LCM_SRCS,
+#     deps = [
+#         "@lcmtypes_robotlocomotion//:lcmtypes_robotlocomotion_java",
+#     ],
+# )
```

Comment:
What is the reason for commenting this out? It would be cleaner to purge this if it is no longer needed.

Reply:
It is an issue finding the package, but anyway it is not necessary. if you prefer I can undo that and fix it, it is not difficult.